rust/stackable-cockpit/src/platform/manifests.rs

```diff
@@ -94,6 +94,7 @@ pub trait InstallManifestsExt {
                         helm_chart.name, helm_chart.version
                     );
 
+                    // Assumption: that all manifest helm charts refer to repos not registries
```

Comment:
note: It might make sense to revisit this feature in the future and unify the handling of helm operations independent of OCI registries or index.yaml-based repositories.

Reply:
Yes, this is out of scope right now but may well be necessary in the future - along with the ability to address a further options with e.g. .tgz charts.